Automattic/pocket-casts-android
# Add effects options to watch app

## Description

This adds playback effects in the Wear OS app.

## Testing Instructions
1. Play an episode from any podcast
2. Navigate to the player screen in the pager
3. ✅ Notice that there is an Effects button at the bottom
4. Tap on the Effects button
5. ✅ Notice that the Effects screen is shown and it matches the Figma design
    (Minus button for speed is not aligned with the speed icon above it as IconButton has a minimum touch size and displays a ripple effect on tap - see timestamp 0.15 in the video)
6. Update speed by tapping plus or minus button for speed
7. ✅ Notice that speed is changed
8. Toggle trim silence toggle button to on
9. ✅ Notice that trim silence slider is shown 
10.  Update trim silence by tapping plus or minus button for trim silence
11. ✅ Notice that trim silence is changed 
12. Toggle trim silence toggle button to off
13. ✅ Notice that trim silence slider is hidden, trim silence is set to off
14. Turn on volume boost
15. ✅ Notice that volume is boosted
16. Restart the app
17. Go to the Effects screen
18. ✅ Notice that values selected earlier are saved

## Screenshots or Screencast 

https://github.com/Automattic/pocket-casts-android/assets/1405144/026fdaf0-4202-4375-a0f9-2cd4da4a1a4e


## Checklist
 
#### I have tested any UI changes...
